upfadapter/pfcp/dispatcher.go

```diff
@@ -64,7 +65,7 @@ func Dispatch(msg *pfcpUdp.Message) {
 				handler.HandlePfcpSessionReportResponse(msg)
 		*/
 	default:
-		logger.PfcpLog.Errorf("unknown pfcp message type [%d] ", msg.PfcpMessage.Header.MessageType)
+		logger.PfcpLog.Errorf("unknown pfcp message type [%d] ", msgType)
```

Comment:
What about something like this?
```suggestion
		logger.PfcpLog.Errorf("unknown pfcp message type [%d]: %s", msgType, msg.MessageTypeName())
```

Reply:
Done